### PR TITLE
[docs] List of references

### PR DESCRIPTION
**Pull request by gsarma**
@tpoterba @jbloom22

### CODE REVIEW

**Comment by tpoterba** [comment]
needs a bump :(

**Comment by tpoterba** [comment]
has a rebase conflict now?

File: ci/ci/constants.py [hunk 1]
```diff
@@ -23,23 +22,4 @@
     'GreatBrando',
     'johnc1231',
     'iitalics'
-}   
-
-profiles = {
```
Comment (tpoterba) [inline ci/ci/constants.py]: er this change wasn't intentional I think? You also got lost the authorized users addition

Comment (gsarma) [reply ci/ci/constants.py]: Yeah, I figured I would just issue a separate PR for that.  Most recent commit to this was 30 minutes ago and removed this section?

Comment (tpoterba) [reply ci/ci/constants.py]: oh, yeah, weird...

Comment (tpoterba) [reply ci/ci/constants.py]: maybe let's try squashing locally and force-pushing? will come show you how

**Pull request closed without merging**